.templates/syncthing/service.yml

```diff
@@ -0,0 +1,19 @@
+  syncthing:
+    image: linuxserver/syncthing:latest
+    container_name: syncthing
+    hostname: raspberrypi #optional
+    
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/Madrid
+    volumes:
+      - ./volumes/syncthing/config:/config
+      
+    ports:
+      - 8384:8384 # Web UI
+      - 22000:22000/tcp # TCP file transfers
+      - 22000:22000/udp # QUIC file transfers
+      - 21027:21027/udp # Receive local discovery broadcasts
```

Comment:
As https://github.com/linuxserver/docker-syncthing/issues/45 points out, local discovery won't work  unless the service is run in host-mode. Synchronizing devices directly on the local LAN, and not using a potentially very slow internet connection to relay data, is very important. 

To enable host-mode add:
```yml
  network_mode: host
```
Of course port-mappings aren't possible when in host-mode, so comment out them. It's good to have them as comments for ease of reference.

Reply:
I read that using host network mode is not the ideal for containers. I was thinking about permforming a first MVP pull request and then improve it step by step (for example in a new issue or pr). What do you think? . Maybe it could be set as an "option"

Reply:
Host mode networking is a restricting option, and shouldn't be used unless needed. It definitely should be used in this case. 

Without it you will get poor local transfer rates:
https://github.com/syncthing/syncthing/blob/main/README-Docker.md#discovery